api/python/quilt3/packages.py

```diff
@@ -947,10 +977,13 @@ def build(self, name, registry=None, message=None):
             registry: registry to build to
                 defaults to local registry
             message: the commit message of the package
+            %(workflow)s
 
         Returns:
             The top hash as a string.
         """
+        registry = get_package_registry(registry)
+        self._validate_with_workflow(registry=registry, workflow=workflow, message=message)
```

Comment:
Here and elsewhere, shouldn't we be documenting the errors that can be thrown in the docstring?

Reply:
Ideally, yes, but it should be well-thought since it becomes the part of the public API, so I'd postpone it until this stuff become more well-defined.